ui/packages/catalog-ui-search/src/main/webapp/react-component/container/metacard-interactions/index.tsx

```diff
@@ -37,8 +36,11 @@ import * as user from '../../../component/singletons/user-instance'
 
 import MarionetteRegionContainer from '../marionette-region-container'
 
+import ExportActions from '../../../react-component/presentation/export-actions'
+
 const Query = require('../../../js/model/Query')
 const wreqr = require('wreqr')
+const Marionette = require('marionette')
```

Comment:
❓ Is wrapping my react component in a `LayoutView` the best way to do this? Or should I handle this differently?

Reply:
You may be able to do the same thing within an ItemView, but I'm not exactly sure which is better

Reply:
It feels odd to import `Marionette` into a react component whether it be with an `ItemView` or `LayoutView`